bedrock/firefox/templates/firefox/family/includes/modules/faq.html

```diff
@@ -0,0 +1,111 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% from "firefox/family/includes/macros.html" import module_tag, browser_border %}
+
+{% set csm_utm_params = '?utm_source=firefox&utm_medium=partnership&utm_campaign=faq' %}
+
+<section id="faq" class="c-faq">
+  <div class="mzp-l-content mzp-t-content-xl">
+    {{ module_tag(title='FAQs', anchor='faq') }}
+    <div class="c-card t-shadow">
```

Comment:
It's a good call-out. In this case, it would be a bigger change though, because I'm using these `c-card` class styles throughout the page. I'm really just borrowing the padding styling and white background on the details answers though, I can pull those out into their own class styles.